### PR TITLE
implement anonymize function for iPv4

### PR DESCRIPTION
**Pull request by larrykimutai**

### CODE REVIEW

File: pii_data.py
Comment (claesmk) [inline pii_data.py:36]: There is no need to change anything here, but, I see that you documented what your regex was doing - which is a very good best practice because they can be so complicated to decipher.  I wanted to point out the [re.VERBOSE](https://docs.python.org/3.9/library/re.html?highlight=subn#re.VERBOSE) option which allows you to document a regex embedded within the regex. Some people do it this way when the string is very complicated.

File: test_pii_data.py
Comment (claesmk) [inline test_pii_data.py:134]: The print() statements are OK but they don't actually test anything.  Here is an example of how I would test one of the cases.  You need to implement something like this so you catch if anonymize is returning the correct value or not. 

```
        # Test anonymize
        self.assertEqual(test_data.has_ipv4(anonymize=True),
                         'Customer Sue Martin contacted the help desk to report an issue '
                         'with their IPv4 address [iPv4 address]')
```

File: pii_data.py
Comment (claesmk) [inline pii_data.py:54]: In the case where you can't find an IPv4 address you should just return the string unmodified.  I'll update the text in the assignment to make this more clear.

File: pii_data.py
Comment (larrykimutai) [reply pii_data.py:36]: Thanks, I read through the documentation and that will be something I will try to implement more

**Review by claesmk — APPROVED** [review]
Approved your changes, however, you might want to try a test case like this for when we get into the next sprint.
```
        test_data = Pii('192.168.1.2 My IP 192.168.1.3 address is 192.168.1.1')  # test an address embedded inside sentence
        self.assertTrue(test_data.has_ipv4())
        # Test anonymize
        self.assertEqual(test_data.has_ipv4(anonymize=True),
                         '[iPv4 address] My IP [iPv4 address] address is [iPv4 address]')
```

**Comment by claesmk** [comment]
Reminder you are waiting for someone else on your team to review your changes - after that you will be able to merge the pull request.

**Pull request merged**
